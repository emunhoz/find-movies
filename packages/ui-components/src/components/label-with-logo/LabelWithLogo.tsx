import React from 'react'
import * as S from './LabelWithLogo.style'

interface ILabelWithLogo {
  backgroundLogo?: string
  logo?: string
  value?: string
}

export const LabelWithLogo = ({ backgroundLogo, logo, value }: ILabelWithLogo) => (
  <S.LabelWithLogo>
    <S.LogoWrapper backgroundLogo={backgroundLogo!}>
      <S.Logo src={logo} alt="logo" width="50" />
    </S.LogoWrapper>
    <S.Value>
      {value}
    </S.Value>
  </S.LabelWithLogo>
)
