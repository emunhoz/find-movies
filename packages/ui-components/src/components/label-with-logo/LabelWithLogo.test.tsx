import React from 'react'
import { render } from '../../utils/test-utils'
import { LabelWithLogo } from './LabelWithLogo'

describe('[Component]: LabelWithLogo', () => {
  test('render the component', () => {
    const { asFragment } = render(<LabelWithLogo />)
    expect(asFragment()).toMatchSnapshot()
  })
})