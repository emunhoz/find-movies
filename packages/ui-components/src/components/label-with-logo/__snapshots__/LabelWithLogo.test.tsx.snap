// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: LabelWithLogo render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ"
  >
    My new LabelWithLogo component example
  </div>
</DocumentFragment>
`;
